### ISSUE TITLE
Is AutoDock-Vina applicable to peptides?

### ISSUE THREAD

**Issue by SoodabehGhaffari** [ccsb-scripps/AutoDock-Vina#302]
Hello,
I was wondering if I can use AutoDock-Vina for docking between proteins and peptides.If not, do you have other suggestions?

Best Regards
Soodabeh

**Comment by rwxayheee** [comment]
Hi @SoodabehGhaffari 
In AutoDock Suite there is a project specialized in docking peptide ligands - [AutoDock CrankPep (ADCP)](https://ccsb.scripps.edu/adcp/). You can compare them method-wise, try a few peptide ligands of your own, visualize the outputs and check the ranking. 

Just a user's opinion here - For short peptides (some 5-9 AA peptide ligands I designed, which barely form a large piece of any secondary structure) I had positive experiences docking them using AutoDock Vina. I got similar top-scored poses in AutoDock Vina and ADCP, but at the moment AutoDock Vina is so much faster. I'm less experienced in docking longer peptides, though.. Maybe you could also ask in a bigger computational chemistry community that are not exclusive to AutoDock :>

**Comment by diogomart** [comment]
I agree with @rwxayheee. As for further suggestions, if you have only standard amino acids, software for protein structure prediction might be useful, e.g. Alphafold, Rosetta, RFdiffusion.
